docs/jwst/skymatch/README.rst

```diff
@@ -14,8 +14,10 @@ cf. :doc:`mrs_imatch step <../mrs_imatch/README>`) of a set of input images
 and computes the signal offsets for each image that will minimize
 the residuals across the entire set in the least squares sence. This comparison
 is performed directly on input images without resampling them onto a common
-grid. By default the sky value computed for each image is recorded, but
-not actually subtracted from the images.
+grid. The overlap regions are computed directly on the sky (celestial sphere)
+for each pair of input images. By default the sky value computed for each image
+is recorded, but not actually subtracted from the images. Also note that the
+meaning of "sky background" is depends on the chosen sky computation method.
```

Comment:
Remove the word "is" (should be 'meaning of "sky background" depends on the chosen ...')